### PR TITLE
TGP-2086: commodity code expired day

### PR DESCRIPTION
**Pull request by vinoocs**

### CODE REVIEW

File: app/controllers/CategorisationPreparationController.scala [hunk 1]
```diff
@@ -143,7 +145,7 @@ class CategorisationPreparationController @Inject() (
   )(implicit
     hc: HeaderCarrier
   ): Future[Done] =
-    if (categorisationInfo.categoryAssessmentsThatNeedAnswers.isEmpty) {
+    if (categorisationInfo.categoryAssessmentsThatNeedAnswers.isEmpty && !categorisationInfo.isCommCodeExpired) {
```
Comment (ChrisCookOC) [inline app/controllers/CategorisationPreparationController.scala:148]: Can test the scenario where commodity code is expired and the categoryAssessmentsThatNeedAnswers is empty to ensure it doesn't go into this case

Comment (vinoocs) [reply app/controllers/CategorisationPreparationController.scala:148]: ah right, had it in mind but missed to add, thanks for spotting @ChrisCookOC :)

Comment (vinoocs) [reply app/controllers/CategorisationPreparationController.scala:148]: sorted, please check now @ChrisCookOC  :)

**Review by ChrisCookOC — DISMISSED** [review]

**Review by Ewan-James-Donovan — DISMISSED** [review]

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by ChrisCookOC** [comment]
See https://github.com/hmrc/trader-goods-profiles-frontend/pull/378

**Pull request closed without merging**